### PR TITLE
Added Sign Up Form In Baluma CSS

### PR DESCRIPTION
**Pull request by B1TSH3LL**
Hey Is there anything else to do with the code. ??? If no please merge the PR and dont forgot to add hacktoberfest and hactoberfest-accepted label to the PR. Thanks :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,5 +20,47 @@ <h1 class="title">
...
+            <p class="title">
+              Already a member?
+            </p>
+            <p class="subtitle">
+              Sign in to continue.
+            </p>
+            <div class="field">
+              <label class="label">Email</label>
+              <div class="control">
+                <input class="input" type="text" placeholder="Email address">
```
Comment (goodmanslaugh) [inline index.html:38]: I don't see `<form>` tag in the code. Please wrap the code inside the form tag.

File: index.html [hunk 2]
```diff
@@ -17,5 +20,47 @@ <h1 class="title">
...
-  </body>
-</html>
+
+  <section class="section">
+    <div class="container">
+      <div class="columns is-centered">
+        <div class="card">
+          <div class="card-content login">
+            <p class="title">
+              Already a member?
```
Comment (goodmanslaugh) [inline index.html:30]: There is no need to have the lines from 29-34

File: index.html [hunk 3]
```diff
@@ -17,5 +20,47 @@ <h1 class="title">
...
+            <div class="field">
+              <label class="label">Email</label>
+              <div class="control">
+                <input class="input" type="text" placeholder="Email address">
+              </div>
+            </div>
+            <div class="field">
+              <label class="label">Password</label>
+              <div class="control">
+                <input class="input" type="email" placeholder="My password">
```
Comment (goodmanslaugh) [inline index.html:44]: This form should have only one input field and the type of input should be text and placeholder should be `@Username`

**Comment by B1TSH3LL** [comment]
Now Please have a look and let me know

**Comment by B1TSH3LL** [comment]
Bruh ????

**Comment by goodmanslaugh** [comment]
Wow this is amazing! LGTM! :rocket: :rocket:

**Comment by B1TSH3LL** [comment]
please add the lables hacktoberfest ,  hacktoberfest2022 , hacktoberfest-accepted 💯

**Comment by B1TSH3LL** [comment]
else my contribution wouldn't be count

**Pull request merged**